addons/fleet/models/fleet_vehicle.py

```diff
@@ -308,3 +320,13 @@ class FleetServiceType(models.Model):
         ('contract', 'Contract'),
         ('service', 'Service')
         ], 'Category', required=True, help='Choose whether the service refer to contracts, vehicle services or both')
+
+
+class FleetVehicleAssignationLog(models.Model):
```

Comment:
This model does not seem to be used anywhere in the code, am I wrong ?

Reply:
After discussion, we should probably coordinate it with the onchange on driver. Otherwise I fear this table will never be populated.